crates/rspack_plugin_runtime/src/runtime_module/get_chunk_filename.rs

```diff
@@ -1,14 +1,15 @@
-use std::{cmp::Ordering, fmt};
+use std::{cmp::Ordering, collections::HashMap, fmt};
```

Comment:
use `rustc_hash::FxHashMap` for better perf

Reply:
Done in 4873c8254da72d798ac0da2599a529abadeb4ae2.